### DISCUSSION TITLE
"Text node" in User Rules' "excludeSelectors"

### DISCUSSION THREAD

**Discussion by iG8R** [immersive-translate/immersive-translate#1468] [Help] [answered]
### 概述 | Summary

Hi.
Is there any way to exclude a `text node` like in the example from the translation?
https://www.69xinshu.com/txt/13823/8196201

![image](https://github.com/immersive-translate/immersive-translate/assets/11407417/73942b87-e8af-44a4-9705-bcd78f44e36c)

**Comment by theowenyoung** [comment, marked as answer]
no, we can't.
